OWML.Common/Menus/IModInputCombinationMenu.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace OWML.Common.Menus
+{
+    public interface IModInputCombinationMenu : IModPopupMenu
+    {
+        event Action OnCancel;
+        event Action<string> OnConfirm;
```

Comment:
Can you swap these so the order is the same as in IModInputCombinationElementMenu?

Reply:
K